hack/update-vendor.sh

```diff
@@ -67,17 +76,17 @@ function ensure_require_replace_directives_for_all_dependencies() {
   go mod edit -json | jq -r ".Replace // [] | sort | .[] | select(${replace_filter})" > "${replace_json}"
 
   # 1a. Ensure replace directives have an explicit require directive
-  cat "${replace_json}" | jq -r '"-require \(.Old.Path)@\(.New.Version)"'             | xargs -L 100 go mod edit -fmt
+  jq -r '"-require \(.Old.Path)@\(.New.Version)"' < "${replace_json}" | xargs -L 100 go mod edit -fmt
```

Comment:
unnecessary cat usage. we can just pipe the file to input. all of the changes on `.*cat.*` lines are this.